app/assets/javascripts/controllers/FileCtrl.coffee

```diff
@@ -1,18 +1,15 @@
-@bruseApp.controller 'FileCtrl', ['FileHandler', '$scope', '$http', (FileHandler, $scope, $http) ->
+@bruseApp.controller 'FileCtrl', ['FileHandler', '$scope', '$rootScope', '$location', '$routeParams', (FileHandler, $scope, $rootScope, $location, $routeParams) ->
   $scope.files = []
   $scope.bruse_files = []
   $scope.new_files = []
   $scope.loading = true
 
-  # get identity information from add view
-  $scope.identity = IDENTITY_PARAMS
```

Comment:
$routeParams collects the varialble from the url, you can see it [here](https://github.com/tkomstadius/bruse/pull/34/files#diff-af5144432cce9b5a21463a4f62507c60R5). The easiest merge would probably be to just skip the $routeParams everywhere and have it like it is in #32.